config/default.json

```diff
@@ -1,5 +1,9 @@
 {
+  "provider": "ses",
```

Comment:
This is the default provider but can be overridden per request. That allows the tests to specify a "mock" provider and will allow e.g. local dev to use the "smtp" provider (which doesn't exist yet).

Reply:
(...and for #6 maybe we'll add a "sendgrid" provider)